gutenberg/acquire/metadata.py

```diff
@@ -271,13 +343,21 @@ def _create_metadata_cache(cache_location):
     """Creates a new metadata cache instance appropriate for this platform.
 
     """
+    try:
+        cache_url = os.getenv('GUTENBERG_FUSEKI_URL', '')
+        return FusekiMetadataCache(cache_location, cache_url)
+    except InvalidCacheException:
+        logging.debug('Unable to create cache based on Apache Jena Fuseki. '
+                      'Next trying BSD-DB implementation.')
```

Comment:
Given that there's no obvious way to turn on debug level for logging, this seems kind of pointless to even have, and also, if a user is seriously trying to use fuseki and it's not working, said user would probably want a warning, though it might be most appropriate to actually throw the InvalidCacheException. They went to the trouble to set an environment variable after all.

Reply:
Good idea. If the environment variable was set, we not throw the exception if the cache can't be instantiated.